outputs.tf

```diff
@@ -55,7 +55,8 @@ output "runner_launch_template_name" {
 
 output "runner_user_data" {
   description = "(Deprecated) The user data of the Gitlab Runner Agent's launch template. Set `var.debug.output_runner_user_data_to_file` to true to write `user_data.sh`."
-  value       = nonsensitive(local.template_user_data)
+  value       = local.template_user_data
+  sensitive   = true
 }
```

Comment:
According to #1019 and changing the `nonsensitive` back and forth, I suggest to remove the whole output. It's deprecated and the value is also written to a local file in the `debug/` directory.